libs/ui-watt/src/lib/foundations/color/_colors.import.scss

```diff
@@ -0,0 +1,103 @@
+    'primary-dark': --watt-primary-colors-primary-dark,
+    // Focus
+    'selection': --watt-focus-colors-selection,
+    'focus': --watt-focus-colors-focus,
+    // Neutrals
+    'black': --watt-neutral-colors-black,
+    'white': --watt-neutral-colors-white,
+    'grey-50': --watt-neutral-colors-grey-50,
+    'grey-100': --watt-neutral-colors-grey-100,
+    'grey-200': --watt-neutral-colors-grey-200,
+    'grey-300': --watt-neutral-colors-grey-300,
+    'grey-400': --watt-neutral-colors-grey-400,
+    'grey-500': --watt-neutral-colors-grey-500,
+    'grey-600': --watt-neutral-colors-grey-600,
+    'grey-700': --watt-neutral-colors-grey-700,
+    'grey-800': --watt-neutral-colors-grey-800,
+    'grey-900': --watt-neutral-colors-grey-900,
+    // States
+    'danger': --watt-state-colors-danger,
+    'danger-light': --watt-state-colors-danger-light,
+    'warning': --watt-state-colors-warning,
+    'warning-light': --watt-state-colors-warning-light,
+    'success': --watt-state-colors-success,
+    'success-light': --watt-state-colors-success-light,
+    'info': --watt-state-colors-info,
+    'info-light': --watt-state-colors-info-light,
+    'contrast': (
+        'primary': --watt-primary-colors-primary-contrast,
+        'primary-light': --watt-primary-colors-primary-light-contrast,
+        'primary-dark': --watt-primary-colors-primary-dark-contrast,
+        // Focus
+        'selection': --watt-focus-colors-selection-contrast,
+        'focus': --watt-focus-colors-focus-contrast,
+        // Neutrals
+        'black': --watt-neutral-colors-black-contrast,
+        'white': --watt-neutral-colors-white-contrast,
+        'grey-50': --watt-neutral-colors-grey-50-contrast,
+        'grey-100': --watt-neutral-colors-grey-100-contrast,
+        'grey-200': --watt-neutral-colors-grey-200-contrast,
+        'grey-300': --watt-neutral-colors-grey-300-contrast,
+        'grey-400': --watt-neutral-colors-grey-400-contrast,
+        'grey-500': --watt-neutral-colors-grey-500-contrast,
+        'grey-600': --watt-neutral-colors-grey-600-contrast,
+        'grey-700': --watt-neutral-colors-grey-700-contrast,
+        'grey-800': --watt-neutral-colors-grey-800-contrast,
+        'grey-900': --watt-neutral-colors-grey-900-contrast,
+        // States
+        'danger': --watt-state-colors-danger-contrast,
+        'danger-light': --watt-state-colors-danger-light-contrast,
+        'warning': --watt-state-colors-warning-contrast,
+        'warning-light': --watt-state-colors-warning-light-contrast,
+        'success': --watt-state-colors-success-contrast,
+        'success-light': --watt-state-colors-success-light-contrast,
+        'info': --watt-state-colors-info-contrast,
+        'info-light': --watt-state-colors-info-light-contrast
+    )
+);
+
+/**
+ * @description
+ *
+ * Get color by name.
+ *
+ * @publicApi
+ */
+@function getColor($name) {
+    $color-var: map-get($_colors, $name);
+    @return var(#{$color-var});
+}
```

Comment:
The reason is to avoid "magic strings" of variables floating across the applications. With the getColor function we have the control as the consumers don't get the variables directly.

Reply:
They are still available/visible in the apps/DevTools. So nothing prevents a feature developer to access these values directly using CSS Custom Properties API (i.e. `var()`).

![image](https://user-images.githubusercontent.com/1096332/135092541-f55fa596-4209-4bba-9dfa-f5c1566d06db.png)

Reply:
That's true, if the consumer wants to break the system they're able to do it because that we're using custom vars, however then they haven't followed the documentation and been creative by themselves. We can't really do anything about that. However this enables us not having "public" variables floating around, so we shouldn't change it more than 2 places if we decide to rename, like you suggest in another comment. Also the setColor function gives us the control of ensuring a color always has a contrast / text color configured as well. Also if I could choose as a developer it's easier for me to remember getColor('primary') than --watt-color-some group-something-something. But I see your point...
 
I could add some extra documentation which states that you shouldn't set the custom vars directly, how about that?
If you're totally against it, I will ofc. change it :)

Reply:
I don't currently see a case where a developer might need contrast colors and the option to set a color. 

That's because the contrast values will be used only when building the Material theme. If we still need to change the style of a certain Material component, we can do that in the specific app using specific selectors.

On the other hand, the `setColor` function will change the value of a CSS Custom Property for a certain selector but then the `getValue` function still has to be used in order to apply the color. e.g.:

![image](https://user-images.githubusercontent.com/1096332/135217517-c9906393-123a-40a2-bae3-b3b8eb1d05ca.png)

Another point is that the developers don't know what color keywords are available for them to use. That can be fixed with documenting them.